satpy/tests/test_composites.py

```diff
@@ -173,83 +175,153 @@ def setUp(self):
         self.ds4 = ds4
 
         # high resolution version - but too big
-        ds4 = xr.DataArray(da.ones((4, 4), chunks=2, dtype=np.float64),
-                           attrs=attrs.copy(), dims=('y', 'x'),
-                           coords={'y': [0, 1, 2, 3], 'x': [0, 1, 2, 3]})
-        ds4.attrs['name'] += '4'
-        ds4.attrs['resolution'] = 500
-        ds4.attrs['rows_per_scan'] = 1
-        ds4.attrs['area'] = AreaDefinition('test', 'test', 'test',
-                                           {'proj': 'merc'}, 4, 4,
-                                           (-2000, -2000, 2000, 2000))
-        self.ds4_big = ds4
+        ds4_big = xr.DataArray(da.ones((4, 4), chunks=2, dtype=np.float64),
+                               attrs=attrs.copy(), dims=('y', 'x'),
+                               coords={'y': [0, 1, 2, 3], 'x': [0, 1, 2, 3]})
+        ds4_big.attrs['name'] += '4'
+        ds4_big.attrs['resolution'] = 500
+        ds4_big.attrs['rows_per_scan'] = 1
+        ds4_big.attrs['area'] = AreaDefinition('test', 'test', 'test',
+                                               {'proj': 'merc'}, 4, 4,
+                                               (-2000, -2000, 2000, 2000))
+        self.ds4_big = ds4_big
 
-    def test_bad_color(self):
+    @pytest.mark.parametrize(
+        "init_kwargs",
+        [
+            {'name': "true_color", "high_resolution_band": "bad", "neutral_resolution_band": "bad"},
```

Comment:
You want two cases in this list. One is the bad high resolution band, the other (a separate dictionary) is the bad neutral resolution band. So a list with two dictionaries in it.

Reply:
Since `name` isn't different between the cases, this can be hardcoded in the test as `name="true_color"` and then followed by `**init_kwargs`.